plutus-emulator/src/Wallet/Emulator/Types.hs

```diff
@@ -486,27 +486,33 @@ validateEm h txn = do
             _ <- put idx'
             pure Nothing
 
+type ServantAction = ReaderT (TVar ServerState) (ExceptT ServantErr IO)
```

Comment:
Again, at this level of abstraction I think we want to avoid talking about servant per se.